python/doc/examples/reliability_sensitivity/reliability/plot_estimate_probability_form_oscillator.py

```diff
@@ -0,0 +1,391 @@
+"""
+Use the FORM - SORM algorithms on non linear function
```

Comment:
"Non linear" is more British English. In US English, one should better use "nonlinear".